relay-profiling/src/android.rs

```diff
@@ -160,8 +178,30 @@ fn parse_profile(payload: &[u8]) -> Result<AndroidProfile, ProfileError> {
     Ok(profile)
 }
 
-pub fn parse_android_profile(payload: &[u8]) -> Result<Vec<u8>, ProfileError> {
-    let profile = parse_profile(payload)?;
+pub fn parse_android_profile(
```

Comment:
This code looks identical to `parse_sample_profile`, is this intentional?

Reply:
It's not identical. `profile` is a different type and we're not setting the `transaction_name` on the `SampleProfile` in `parse_sample_profile`.

I'd be willing to refactor it but I don't see how we can beat the simplicity of this vs making a common function to copy tags on both structs.

Reply:
Oh sorry, I missed that. I agree this is simpler.